app/containers/Header/index.js

```diff
@@ -149,21 +154,45 @@ const SearchWrap = styled(Box)`
   }
 `;
 
-const renderPages = ({ match, onClick, align }) =>
+const navButtonOnClick = ({ match, onClick, align, locale }) =>
   PAGES &&
   Object.values(PAGES)
     .filter(page => page.primary)
-    .map(page => (
-      <ButtonNavPrimary
-        key={page.key}
-        active={page.key === match}
-        disabled={page.key === match}
-        align={align}
-        onClick={() => onClick(page.key)}
-      >
-        <FormattedMessage {...rootMessages.page[page.key]} />
-      </ButtonNavPrimary>
-    ));
+    .map(page =>
+      page.key === 'download' ? (
+        <a
+          href={XPATHS.download[locale] || XPATHS.download[DEFAULT_LOCALE]}
+          target="_blank"
+          rel="noopener noreferrer"
+          key={page.key}
+        >
+          <ButtonNavPrimary>
```

Comment:
one more note: I do not think nesting a `button` inside an `a` is valid HTML - will fix

Reply:
just use `as="a"` attribute on the button itself